core/templates/dev/head/pages/exploration_editor/editor_navigation_directive.html

```diff
@@ -90,7 +90,7 @@
   </li>
 
   <div ng-if="userIsLoggedIn" class="nav navbar-nav oppia-navbar-nav oppia-help-dropdown pull-right ng-cloak">
-    <li uib-dropdown class="uib-dropdown" popover-is-open="popoverControlObject.postTutorialHelpPopoverIsShown"
+    <li uib-dropdown class="uib-dropdown uib-popover" popover-is-open="popoverControlObject.postTutorialHelpPopoverIsShown"
```

Comment:
Just to check: this has a uib-dropdown class, but the one in exploration_footer_directive.html does not. Is there a reason for the difference? (Just wondering if this one was redundant.)

Reply:
Actually while I was making the upgrade I noticed that in many places this same class was used for styling. The new dropdown component uses a directive and not a css class. So in 
places where the dropdown class was defined I added it as the directive and retained the class so as not to create any styling issues. Though some of them would be redundant. Even the uib-popover class I added because it was styled using that class.

Reply:
Ah, OK. I would advise getting rid of the ones that are redundant, if possible. It makes the code easier to maintain in the future.

Reply:
ok i'll do that then 👍